.github/workflows/CI.yml

```diff
@@ -13,14 +13,17 @@ jobs:
       fail-fast: false
       matrix:
         group:
+          #fixes 682
+          -ODEBPINN
```

Comment:
`- ODEBPINN`

needs a space. One check runs for each group, not for each testset